src/client/wieldmesh.h

```diff
@@ -58,6 +60,19 @@ class ItemMeshBufferInfo
 
 	ItemMeshBufferInfo(const TileLayer &layer);
 
+	ItemMeshBufferInfo(AnimationInfo *animation,
+			bool override = false, video::SColor color = {}) :
```

Comment:
ideally call this something other than `override`, because that's a (soft-ish) keyword (for example you can see how the syntax highlighting here on GH is off). best to avoid trouble.